src/components/Lists/List.vue

```diff
@@ -2,9 +2,9 @@
   <div :class="['list', viewClass, { unique: unique && view !== 'masonry' }]">
     <header>
       <span class="list-name">
-        <i
+        <sort-icon
           v-if="autoSortEnabled"
-          class="fas fa-magic"
+          :sortOrder="list[listIndex].sortOrder"
```

Comment:
params are kebab case, it's a [vue thing](https://vuejs.org/v2/guide/components-props.html) 
```suggestion
          :sort-order="list[listIndex].sortOrder"
```